service/history/api/consistency_checker.go

```diff
@@ -226,80 +264,44 @@ func (c *WorkflowConsistencyCheckerImpl) getWorkflowLeaseValidatedByCheck(
 	return NewWorkflowLease(wfContext, release, mutableState), nil
 }
 
-func (c *WorkflowConsistencyCheckerImpl) getCurrentWorkflowContext(
+func (c *WorkflowConsistencyCheckerImpl) getWorkflowLeaseValidatedByClock(
```

Comment:
looks like this method is no longer being used and can be removed.

Reply:
removed